libfwupd/fwupd-enums.h

```diff
@@ -79,6 +79,7 @@ typedef enum {
  * @FWUPD_DEVICE_FLAG_INSTALL_PARENT_FIRST:	Install composite firmware on the parent before the child
  * @FWUPD_DEVICE_FLAG_IS_BOOTLOADER:		Is currently in bootloader mode
  * @FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG:		The hardware is waiting to be replugged
+ * @FWUPD_DEVICE_FLAG_FORCE:			Always ignore safety checks on this device
```

Comment:
I think I know what you're trying to achieve, but `FORCE` isn't the same as `RELAX_CHECKS`-- but that seems a very big dangerous hammer. I'd much rather see this broken out like `FWUPD_DEVICE_FLAG_IGNORE_MODEL_ID` or `FWUPD_DEVICE_FLAG_IGNORE_VERSION_LOWEST` -- discussion certainly welcome.

Reply:
I don't disagree it's big and dangerous. These are two different problems that I at least thought it should be same solution.

With the uefi case force will come on command line argument so I do think that way I did it is right.

The Thunderbolt case validation can fail for a variety of reasons and I'm looking for a catch all when I know I want to ignore failures. Right now it might be model ID but what happens if something else fails in drom checks later?
Maybe relax checks is what I'm looking for. The update method can look for either force or that device flag rather than daemon lumping them together.